modules/Bio/EnsEMBL/Taxonomy/PipeConfig/ImportNCBItaxonomy_conf.pm

```diff
@@ -98,153 +100,184 @@ sub resource_classes {
+      -module     => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+      -parameters => {
+                       inputfile => '#scratch_dir#/merged.dmp',
+                       delimiter => "\t\Q|\E\t?",
+                     },
+      -flow_into  => {
+                       1 => [ 'web_name_patches' ],
+                       2 => { '?table_name=ncbi_taxa_name' =>
+                              { 'name' => '#_0#',
+                                'taxon_id' => '#_1#',
+                                'name_class' => 'merged_taxon_id'
+                              }
+                            },
+                      },
+    },
+    {
+      -logic_name    => 'web_name_patches',
+      -module        => 'Bio::EnsEMBL::Hive::RunnableDB::DbCmd',
+      -parameters    => {
+                          input_file => $self->o('base_dir').'/ensembl-taxonomy/sql/web_name_patches.sql',
+                        },
+    },
+    {
+      -logic_name => 'add_import_date',
+      -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+      -parameters => {
+                       sql => 'INSERT INTO ncbi_taxa_name (taxon_id, name_class, name) SELECT taxon_id, "import date", CURRENT_TIMESTAMP FROM ncbi_taxa_node WHERE parent_id=0 GROUP BY taxon_id',
+                     },
+      -flow_into  => { 1 => [ 'cleanup' ], },
+    },
+    {
+      -logic_name => 'cleanup',
+      -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+      -parameters => {
+                       cmd => 'rm -rf #scratch_dir#',
+                     },
+      -flow_into  => { 1 => [ 'post_load_checks' ], },
+    },
+    {
+      -logic_name => 'post_load_checks',
+      -module     => 'Bio::EnsEMBL::Taxonomy::RunnableDB::PostLoadChecks',
+      -parameters => {
+                       tgt_host => $self->o('tgt_host'),
+                       copy_to_tgt_host => $self->o('copy_to_tgt_host'),
+                     },
+      -flow_into  => { 1 => WHEN(
+                                    'defined #tgt_host# && #copy_to_tgt_host#' => [ 'copy_database' ],
+                                ),
+                     },
+    },
+    {
+      -logic_name => 'copy_database',
+      -module     => 'ensembl.production.hive.ProductionDBCopy',
+      -language   => 'python3',
+      -rc_name    => 'default',
+      -parameters => {
+                       endpoint => $self->o('copy_service_uri'),
+                       payload  => $self->o('payload'),
+                       method   => 'post',
+                    },
+      -flow_into  => { 1 => WHEN(
+                            'defined #metadata_host_uri# && #metadata_db_name#'  => ['update_taxonomy_node_table_in_metadata_db']
+                        ),
+      },
+    },
+    {
+      -logic_name => 'update_taxonomy_node_table_in_metadata_db',
+      -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+      -parameters => {
+                       db_conn => $self->o('metadata_host_uri').$self->o('metadata_db_name'),
```

Comment:
We should use `INSERT INTO` with a `on duplicate key` statement. 
In order to do this we need to add some keys on one of the table as well.

Reply:
we are moving away from this approach to insert the non duplicate records into tables